libs/browser-wallet/src/frontend/routes/auth/settings/home/sections/export-recovery-phrase-section/export-recovery-phrase-section.tsx

```diff
@@ -15,11 +16,24 @@ export const locators = {
 export const ExportRecoveryPhraseSection = () => {
   const [open, setOpen] = useState(false);
   const [recoveryPhrase, setRecoveryPhrase] = useState<string | null>(null);
+  const { request } = useJsonRpcClient();
+  useEffect(() => {
```

Comment:
non blocking: I would prefer this was run on an action, such as the click to open the dialog